packages/lib/src/Paywall.wc.svelte

```diff
@@ -3,6 +3,7 @@
 <script lang="ts">
   import Base from './Base.svelte';
   import type { IconName } from './icons/types';
+  import type { Translate } from './i18n';
```

Comment:
Export / import this as "Translation" or "Translations".

I don't think it's good practice for a type definition to be a verb.